test/node/conversation-direct.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let user;
+let client2;
+let user2;
+let conversation;
+describe('Direct Conversation', () => {
+
+    before(async () => {
+        client = new Circuit.Client(config.bot1);
+        user = await client.logon();
+        client2 = new Circuit.Client(config.bot2);
+        user2 = await client2.logon();
+    });
+
+    after(async () => {
+        await client.logout();
+        await client2.logout();
+    });
+
+    it('should create a direct conversation', async () => {
+        const res = await client.createDirectConversation(user2.userId);
+        conversation = res.conversation;
+        conversation.participants.forEach(userId => {
+            if (!(userId === user.userId || userId === user2.userId)) {
```

Comment:
if (userId !== user.userId && userId !== user2.userId) {

or better use the includes

if (conversation.participants.includes(user.userId) && conversation.participants.includes(user2.userId)) {

Reply:
assert(conversation.participants.includes(user.userId) && conversation.participants.includes(user2.userId));